Update short-lived-certificates-legacy.mdx (#22469)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/applications/non-http/short-lived-certificates-legacy.mdx

### Connect through a browser-based terminal

> End users can connect to the SSH session without any configuration by using Cloudflare's browser-based terminal. By default, the user will be prompted for a username/password login. If you would like to use certificate based authentication, create a short-lived certificate for the specific Access Application configured for Browser-Rendered SSH. The SSH certificate can be configured at **Access** > **Service Auth** > **SSH**  To enable, refer to [Browser-rendered terminal](/cloudflare-one/applications/non-http/browser-rendering/).

Reviewer: ```suggestion
End users can connect to the SSH session without any configuration by using Cloudflare's browser-based terminal. To enable, refer to [Browser-rendered terminal](/cloudflare-one/applications/non-http/browser-rendering/).

By default, the browser-based terminal prompts the user for a username/password login. If you would like to use certificate based authentication, make sure you have [created a short-lived certificate](#3-generate-a-short-lived-certificate-public-key) for the specific Access application configured for browser-rendered SSH.
```
